frontend/src/features/admin/users/ManageUsers.tsx

```diff
@@ -119,29 +170,38 @@ export const ManageUsersPage = () => {
         }}
       />
       {
-        <TableContainer fluid>
-          <Table<IUserRecord>
-            name="usersTable"
-            columns={columns}
-            pageIndex={pageIndex}
-            data={userList}
-            defaultCanSort={true}
-            pageCount={Math.ceil(pagedUsers.total / pageSize)}
-            pageSize={pageSize}
-            onRequestData={onRequestData}
-            onSortChange={(column, direction) => {
-              if (!!direction) {
-                dispatch(getUsersSortAction({ [column]: direction }));
-              } else {
-                dispatch(getUsersSortAction({}));
-              }
-            }}
-            sort={sort}
-            onPageSizeChange={size => dispatch(setUsersPageSize(size))}
-            loading={!(users && !users.isFetching)}
-            clickableTooltip="Click IDIR/BCeID link to view User Information page"
-          />
-        </TableContainer>
+        <>
+          <Ribbon>
+            <TooltipWrapper toolTipId="export-to-excel" toolTip="Export to Excel">
```

Comment:
in other tables we have "export to excel" and "export to CSV", not sure if it matters if we have both?